src/components/Router.js

```diff
@@ -0,0 +1,14 @@
+import {Routes, Route} from "react-router-dom";
+import Home from "../pages/home";
+import { Instructions } from "../pages/instructionPage";
+
+export const Router = () => {
+    return (
+        <Routes>
+            <Route path="/" element={<Home />}/>
+            <Route path="/instructions" element={<Instructions />}/>
+            <Route path="/instructions" element={<Instructions />}/>
```

Comment:
why there are two routes for the same path.

Reply:
Thanks for pointing it out, I forgot to remove that line.